packages/core/src/actors/callback.ts

```diff
@@ -63,10 +63,10 @@ export function fromCallback<TEvent extends EventObject, TInput = unknown>(
   const logic: CallbackActorLogic<TEvent, TInput> = {
     config: invokeCallback,
     start: (_state, { self, system }) => {
-      system._relay(self, self, { type: XSTATE_INIT });
```

Comment:
this was *super* confusing because `XSTATE_INIT` is a special event and yet this "custom" logic was sending the same type to itself manually